app/controllers/api/v1/mappings_controller.rb

```diff
@@ -27,6 +27,23 @@ def create
     render json: mapping, include: :specification
   end
 
+  ###
+  # @description: Creates the mapping terms. This method is used when the user has already
+  #   decided which terms to map to the spine
+  ###
+  def create_terms
+    # Validate mapping existence (We will create mapping terms for an existent mapping)
+    mapping = Mapping.find(params[:mapping_id])
+    terms = params[:terms]
+
+    raise ActiveRecord::RecordInvalid.new("Mapping terms were not provided") unless terms.count.positive?
```

Comment:
Not a biggie, but using `Enumerable#any?` is a shorter way of checking whether a collection has any items.